src/trigger-circleci-workflows.ts

```diff
@@ -61,17 +73,58 @@ async function triggerCircleCI(
         );
       }
     });
-  context.log.info(
-    {data},
-    `Build triggered successfully for ${context.payload['pull_request']['html_url']}`
-  );
+  context.log.info({data}, `Build triggered successfully for ${repoKey}`);
 }
 
 export function circlePipelineEndpoint(repoKey: string): string {
   return `/api/v2/project/github/${repoKey}/pipeline`;
 }
 
-async function runBot(context: probot.Context): Promise<void> {
+export function genParametersForPR(
+  config: object,
+  context: probot.Context,
+  appliedLabels: string[]
+): object {
+  context.log.debug({config, appliedLabels}, 'genParametersForPR');
+  const parameters = {};
+  const labelsToParams = config['labels_to_circle_params'];
+  for (const label of Object.keys(labelsToParams)) {
+    // ci/all is a special label that will set all to true
+    if (appliedLabels.includes('ci/all') || appliedLabels.includes(label)) {
+      parameters[labelsToParams[label].parameter] = true;
+    }
+  }
+  return parameters;
+}
+
+function genParametersForPush(config: object, context: probot.Context): object {
```

Comment:
Not exactly sure what "parameters" here means without reading this function. The output type (`object`) isn't very descriptive either. I'm going to read the object body now.

Reply:
Ah, CircleCI parameters, now I see.

Reply:
Any chance for a more descriptive type?

Reply:
This could probably be renamed to `genCircleParamatersForPush` to make it a bit more descriptive.

As for the return type, an object is specified since it's easier to get working with the `data` input for the eventual circleci API call, I had attempted to make this a `Map<string, boolean>` earlier and had failed.